### DISCUSSION TITLE
Draw.io Markdown problem with links

### DISCUSSION THREAD

**Discussion by seba300** [requarks/wiki#5244] [Error / Bug Report]
When I create diagram in draw.io on markdown page and I wanna add to the blocks link properties, everything works great. I can touch block and he redirect me to another page, but when I also try to add a text to this blocks, every block redirect me to only one page or the blocks are disabled.

**Comment by strg-v** [comment]
Same issue here. I tried to add different links to different objects. I only get redirected to the link attached to the object, which is located at the top hierarchical level.
